conf/switches.conf.example

```diff
@@ -6,6 +6,7 @@
 # http://www.fsf.org/licensing/licenses/gpl.html
 
 [default]
+type=Cisco::Catalyst_2960
```

Comment:
There should be...

Switches will always extend 'default'

There should even have been one already in my opinion.

Users are already setting one anyway, I'm just defining one so there is no need to handle the case where there is none and make the code heavier just for a config parameter.

I'm open to change it but that will require more arguments than a simple 'should not'